components/consumers/dependency-track/task.yaml

```diff
@@ -21,13 +22,18 @@ spec:
     - name: consumer-dependency-track-owner-annotation
       type: string
       default: "Owner"
+    - name: consumer-dependency-track-debug
+      type: string
+      default: "false"
   workspaces:
     - name: output
       description: The workspace containing the source-code to scan.
   steps:
   - name: run-consumer
-    imagePullPolicy: IfNotPresent
-    image: '{{ default "ghcr.io/ocurity/dracon" .Values.image.registry }}/components/consumers/dependency-track:{{ .Chart.AppVersion }}'
+    imagePullPolicy: Always
+    # image: '{{ default "ghcr.io/ocurity/dracon" .Values.image.registry }}/components/consumers/dependency-track:{{ .Chart.AppVersion }}'
```

Comment:
Maybe add a ToDO here and a ticket, to have a reminder of where the image is hard-coded.